src/models/event.ts

```diff
@@ -685,14 +686,6 @@ export class MatrixEvent extends TypedEventEmitter<MatrixEventEmittedEvents, Mat
      * attempt is completed.
      */
     public async attemptDecryption(crypto: Crypto, options: IDecryptOptions = {}): Promise<void> {
-        // For backwards compatibility purposes
```

Comment:
Should this qualify as a breaking change even though the signature is unchanged?

Reply:
The type didn't allow for the backwards compatibility so it got partially removed at some point in the past